QuickSort.c

```diff
@@ -7,6 +7,14 @@ void swap(int * i, int * j)
     *i = *j;
     *j = temp;
 }
+void readData(int arr[], int n)

```

Comment:
This probably needs a new line before the definition of the function. It would really be cool to use a linter. A quick Google search revealed this: https://atom.io/packages/linter-gcc, but please feel free to explore. The point of the assignment is not for you to go through all the cases and fix it by hand. Imagine doing that for a couple of thousand files? Impossible... The point is for you to explore the ecosystem of the programming language that you're working with... 😸

Reply:
Just try and run it from the command line. It doesn't have to be integrated in the editor. Or try a different editor. Sublime, or Atom, or VS Code.